qemu/tests/bridge_qinq.py

```diff
@@ -304,7 +304,7 @@ def compare_host_guest_md5sum(name):
                              host_path, guest_path)
         if not compare_host_guest_md5sum(host_path):
             test.fail("md5sum mismatch on guest and host")
-
+        session.cmd("rm -rf %s" % guest_path)
```

Comment:
Same for the following 2 lines,  otherwise those three steps will be skipped when test fail.

Reply:
Hi @yanan-fu Thanks for your reminder. I have updated. please help me review it again. thanks a lot.